x-pack/plugins/ingest_manager/server/saved_objects/index.ts

```diff
@@ -218,8 +219,15 @@ const savedObjectTypes: { [key: string]: SavedObjectsType } = {
           },
         },
         revision: { type: 'integer' },
+        updated_on: { type: 'keyword' },
+        updated_by: { type: 'keyword' },
+        created_on: { type: 'keyword' },
```

Comment:
Why not using the type date?

Reply:
Good point @nchaulet . I copied the definition from the Agent Config SO which had `keyword`, but now see that Enrollment API Keys SO does use `date`. I will change it.
Thanks 👍